### ISSUE TITLE
[Button]: Wrong colors in HC theme

### ISSUE THREAD

**Issue by bmdalex** [microsoft/fluent-ui-react#1300]
### Description

`Button` `primary` has wrong colors in HC theme.

#### Expected:
- `color: black`
- `backgroundColor: white`

![image](https://user-images.githubusercontent.com/828692/57274710-e862e280-7050-11e9-82b2-2ef42f044203.png)

#### Actual:
Colors are swapped:
- `color: white`
- `backgroundColor: black`

![Screenshot 2019-05-07 at 15 50 14](https://user-images.githubusercontent.com/5442794/57304778-412d8c00-70e0-11e9-8b64-8ece83207ca0.png)

#### Expected:
- `backgroundColor: cyan`

#### Actual:
- `backgroundColor: yellow`

### Note:

In general to keep things consistent for HC, elements should get:

- Focused background - cyan
- Hover background - yellow
- Focus/hover foreground - black
- Focus border - 1px cyan border, 1px black border
_(https://github.com/stardust-ui/react/pull/1269#discussion_r281469392)_

_Originally posted by @codepretty in https://github.com/stardust-ui/react/pull/1269/files_

**Comment by bmdalex** [comment]
@codepretty @bcalvery @notandrew to confirm

**Comment by codepretty** [comment]
Yes, this seems right to me. One thing to note is that the cyan focus background color should be for both primary and secondary buttons, while the white background color for at rest buttons should only apply to primary.

![image](https://user-images.githubusercontent.com/828692/57330030-cc4f5780-70c9-11e9-95f8-b2ad7a21c320.png)
